recipes/ocilib/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.15)
+project(test_package LANGUAGES CXX)
+
+set(CMAKE_CXX_STANDARD 11)
+
+find_package(ocilib REQUIRED CONFIG)
+
+add_executable(${PROJECT_NAME} test_package.cpp)
+target_link_libraries(${PROJECT_NAME} PRIVATE ocilib::ocilib)
```

```suggestion

find_package(ocilib REQUIRED CONFIG)

add_executable(${PROJECT_NAME} test_package.cpp)
target_link_libraries(${PROJECT_NAME} PRIVATE ocilib::ocilib)
target_compile_features(${PROJECT_NAME} PRIVATE cxx_std_11)
```

Prefer CMake target features